libraries/app/database_api.cpp

```diff
@@ -258,6 +258,35 @@ class database_api_impl : public std::enable_shared_from_this<database_api_impl>
          });
          return result;
       }
+      vector<limit_order_object> get_limit_orders(const asset_id_type& a, const asset_id_type& b, const uint32_t& limit)const
```

Comment:
Using `const &` for these parameters are not necessary, they are simple types or tiny objects, better to pass by value.